packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -127,6 +127,11 @@ foreach(bundled_library ${PLUGIN_BUNDLED_LIBRARIES})
     COMPONENT Runtime)
 endforeach(bundled_library)
 
+set(NATIVE_ASSETS_DIR "${PROJECT_BUILD_DIR}native_assets/linux/")
+install(DIRECTORY "${NATIVE_ASSETS_DIR}"
+   DESTINATION "${INSTALL_BUNDLE_LIB_DIR}"
+   COMPONENT Runtime)
+
```

Comment:
This change will need a migration (see [here](https://github.com/flutter/flutter/tree/main/packages/flutter_tools/lib/src/migrations) and [here](https://github.com/flutter/flutter/blob/main/packages/flutter_tools/lib/src/linux/build_linux.dart#L46)), otherwise packages using this functionality will just fail to work in all existing Flutter apps.